lib/ansible/modules/cloud/amazon/ec2_elb_facts.py

```diff
@@ -80,6 +80,14 @@
 
 import traceback
 
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.ec2 import (
```

Comment:
Is there an established Ansible style that says this should be kept on one line? (honest question, I am new to the codebase)
In general I prefer putting things like this into multiple lines as it's a lot clearer later on in diffs when elements are added or removed, and overall just easier (IMO) to read.